### ISSUE TITLE
Jars missing version number in lib

### ISSUE THREAD

**Issue by DanVanAtta** [triplea-game/triplea#317]
Introduced to master with PR #201: lib/tritonus_share.jar, does not have a version number.

upnp.jar also does not have a version number. 

The rest of the jars have version numbers, examples:
```
 laf-plugin-7.3.jar    substance-7.3-tools.jar
derby-10.10.1.1.jar          laf-widget-7.3.jar    substance-flamingo-7.3.jar
flamingo-7.3.jar             mailapi-1.5.1.jar     substance-swingx-7.3.jar
```

**Comment by DanVanAtta** [comment]
Plan is to remove the Jars add in #201 anyways.. upnp.jar I suppose can be grandfathered in. Once the new jars are dealt with, this can be closed. I'll be taking care of the Jar removal from 201, assigning to myself.

**Comment by DanVanAtta** [comment]
Can be closed once #418 is merged
